meta-mender-tegra/recipes-mender/tegra-state-scripts/files/redundant-boot-install-script-uboot

```diff
@@ -26,22 +26,27 @@ if [ $num_slots != 2 ]; then
         exit 1
     fi
 fi
-current_slot=`nvbootctrl get-current-slot`
-echo "Current boot slot: $current_slot"
 
-new_boot_part=`fw_printenv -n mender_boot_part`
-mnt=`mktemp -d -t nvbup.XXXXXX`
+current_boot_part=$(mount | grep ' / ' | cut -d' ' -f 1)
```

Comment:
I think this can fail for some more complicated setups where rootfs is an overlay, not sure whether we need to support this.  Here's a way to do a simmilar thing with stat: https://github.com/OE4T/meta-tegra/issues/143#issuecomment-509917250